.github/config/ct.yaml

```diff
@@ -12,4 +12,5 @@ excluded-charts:
   - common
   - hashicorp=https://helm.releases.hashicorp.com
   - bitnami=https://charts.bitnami.com/bitnami
+  - feast-ui
```

Comment:
I think this line makes the `test` workflow fail

Reply:
I added this line in order to prevent the chart test for running for this chart. It's not possible for the test to pass at the moment because the feast-ui image is not on a public image repository.

Reply:
@shydefoo how can we completely skip the test step for a chart?

Reply:
We can try to set the target branch in the temporarily to this branch
```
target-branch: add-feast-ui-chart
```